Resources/public/css/theme/views/universaldiscovery/selected.css

```diff
@@ -105,18 +107,21 @@
 
 .ez-view-universaldiscoveryselectedview .ez-ud-selected-contenttype {
     margin-top: 0.5em;
-    margin-bottom: 0.5em;
+    margin-bottom: 0.15em;
 }
 
 .ez-view-universaldiscoveryselectedview .ez-ud-selected-infos {
     margin-left: 2em;
+    margin-top: 1.5em;
 }
 
 .ez-view-universaldiscoveryselectedview .ez-ud-selected-infos .ez-ud-selected-infos-title {
     font-weight: bold;
+    font-size: 0.8em;
 }
 
 .ez-view-universaldiscoveryselectedview .ez-ud-selected-infos .ez-ud-selected-info {
     margin-bottom: 0.5em;
     margin-left: 0;
+    font-size: 0.8em;
```

Comment:
to not repeat that, what about defining the font-size on `.ez-ud-selected-infos` ?